src/app/pages/visitsCalendar/visitsCalendar.component.ts

```diff
@@ -107,23 +107,48 @@ export class VisitsCalendarComponent implements OnInit {
         });
     }
 
+    private slotToVisitEvent(slot: TimeSlot): VisitEvent{
+        return {
+            slotId: slot.id,
+            title: "Visit",
+            start: new Date(slot.startDateTime),
+            end: new Date(slot.endDateTime),
+            color: this.colors.blue,
+            availableForSelfSign: slot.availableForSelfSign
+        };
+    }
+
+    private doSomethingWojtekKnowsWhat([event, appointment]: [VisitEvent, Appointment]): VisitEvent{
```

Comment:
Filtruje terminy których użytkownik nie może widzieć i przydziela im kolorki w zależności od zajętości xD

Do nieczytelnego kodu się przyznale i olewam system